app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+  
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %><br />
+  </div>
+
+  <div class="field">
+    <%= f.label :photo %><br />
+    <%= f.text_field :photo %><br />
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_area :bio %><br />
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
```

Comment:
- [ ] The registration page should match the design with the wireframe. Please update the devise registration view.

### Your design 
![Screenshot from 2021-12-08 21-36-34](https://user-images.githubusercontent.com/22434660/145242126-1aa21b26-58d7-4197-ba1f-959d2874499d.png)


### Given design (pasting the login page but design should be similar with this

![Screenshot from 2021-12-08 21-38-17](https://user-images.githubusercontent.com/22434660/145242315-41364541-6812-45f7-a721-4db2299c881d.png)